src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -162,6 +162,12 @@ public string Path
             }
         }
 
+        /// <summary>
+        /// Returns the timestamp at which the next document expiration should take place, <c>0</c>
+        /// value will be returned if there are no documents with expiration times.
+        /// </summary>
+        public ulong NextDocExpiration => Native.c4db_nextDocExpiration(_c4db);
```

Comment:
Not related to this PR.  I guess it didn't fully get untangled, so remove it from here.